pkgs/development/compilers/dotnet/print-hashes.sh

```diff
@@ -46,11 +46,117 @@ platform_sources () {
+      "runtime.linux-musl-arm64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.linux-musl-arm64.Microsoft.NETCore.DotNetHost" \
+      "runtime.linux-musl-arm64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.linux-musl-arm64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.linux-musl-arm.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.linux-musl-arm.Microsoft.NETCore.DotNetHost" \
+      "runtime.linux-musl-arm.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.linux-musl-arm.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.linux-musl-x64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.linux-musl-x64.Microsoft.NETCore.DotNetHost" \
+      "runtime.linux-musl-x64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.linux-musl-x64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.linux-x64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.linux-x64.Microsoft.NETCore.DotNetHost" \
+      "runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.osx-arm64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.osx-arm64.Microsoft.NETCore.DotNetHost" \
+      "runtime.osx-arm64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.osx-arm64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.osx-x64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.osx-x64.Microsoft.NETCore.DotNetHost" \
+      "runtime.osx-x64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.osx-x64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.win-arm64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.win-arm64.Microsoft.NETCore.DotNetHost" \
+      "runtime.win-arm64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.win-arm64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.win-arm.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.win-arm.Microsoft.NETCore.DotNetHost" \
+      "runtime.win-arm.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.win-arm.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.win-x64.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.win-x64.Microsoft.NETCore.DotNetHost" \
+      "runtime.win-x64.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.win-x64.Microsoft.NETCore.DotNetHostResolver" \
+      "runtime.win-x86.Microsoft.NETCore.DotNetAppHost" \
+      "runtime.win-x86.Microsoft.NETCore.DotNetHost" \
+      "runtime.win-x86.Microsoft.NETCore.DotNetHostPolicy" \
+      "runtime.win-x86.Microsoft.NETCore.DotNetHostResolver" \
+    )
+
+    local nuget_url="$(curl -f "https://api.nuget.org/v3/index.json" | jq --raw-output '.resources[] | select(."@type" == "PackageBaseAddress/3.0.0")."@id"')"
+
+    for pkg in "${pkgs[@]}"; do
+        local url hash
+        url="${nuget_url}${pkg,,}/${version,,}/${pkg,,}.${version,,}.nupkg"
+        hash="$(nix-prefetch-url "$url")"
+        echo "      (fetchNuGet { pname = \"${pkg}\"; version = \"${version}\"; sha256 = \"${hash}\"; })"
```

Comment:
Can we use `nuget-to-nix` for this? Maybe it could accept a package list instead of just a nuget package directory.

Reply:
Hmm, I don't think adding a package list to nuget-to-nix is neccessary. It wouldn't be used anywhere else. Its simpler to just have this specific generation script here.

Reply:
I just brought it up for avoiding code repetition, and in case someone may use a custom version of the SDK, with the dependency packages being in a private nuget repo. This script would break, but `nuget-to-nix` can handle that just fine.

I don't think that's a blocking issue though, since we only package the official SDK.